software/ente.yml

```
@@ -0,0 +1,11 @@
+name: "Ente"
+website_url: "https://ente.io/"
+source_code_url: "https://github.com/ente-io/ente"
+description: "An end-to-end encrypted photo-sharing platform (alternative to Google Photos, Apple Photos)."
+licenses:
+  - AGPL-3.0
+platforms:
+  - Go
+  - Docker
```

```suggestion
platforms:
  - Nodejs
```

https://help.ente.io/self-hosting/guides/web-app only mentions a npm-based installation method.

Also, is this all that is required to run a fully working service? It looks like this only sets up the web front-end.

---

https://github.com/ente-io/ente/blob/main/server/docs/docker.md
As far as I can see, this would be the installation with docker

And I thought this would be the go way?
https://github.com/ente-io/ente/blob/main/server/RUNNING.md#running-without-docker